sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -194,6 +210,10 @@ export interface AssessmentModel {
 }
 
 export interface CalculatedAssessmentModel {
+  calculatedDataAdditionalTransportKm: number;
+  calculatedDataAdditionalTransportCost: number;
+  calculatedDataAdditionalTransportWeeks: number;
+  calculatedDataAdditionalTransportPlacement: "yes" | "no";
```

Comment:
We have been using in many places the `YesNoOptions`, for instance for `studentDataAdditionalTransportPlacement`. Is there a reason not to use it here also?
I do not recall if there is an agreement to keep this object with primitive types only but I do not see a reason to not type these ones.

Reply:
Was this comment accepted? I can't see the change. Please let me know if I am missing something or if the comment will not be implemented for some reason.

Reply:
It is updated now.